.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
+  curly: [2, multi-line]
+  eqeqeq: 2
+  indent: [2, tab, { SwitchCase: 0 }]
+  linebreak-style: [2, unix]
+  object-curly-spacing: [2, never]
+  semi: [2, always]
+  space-after-keywords: [2, always]
+  space-before-function-paren: [2, never]
+  spaced-comment: [2, always]
+  no-console: 0
+  no-irregular-whitespace: 2
```

Comment:
This is supposed to be in `eslint:recommended`, yet you have to specify it otherwise ESLint doesn't catch irregular whitespaces in our codebase. Do you happen to know why?! Are there any other rules that are supposed to be there but are ignored?
I am going to report this as an issue to ESLint, unless you tell me there is a good reason for this.